### PR TITLE
Remove Font Awesome References

### LINKED ISSUE #4450: Strip unnecessary CSS from source interface
The CSS served via the *Source Interface* currently includes the full [Font Awesome Free 5.0.6](https://fontawesome.com/) styling. As the name suggests, Font Awesome is an icon font; the purpose of the CSS styles is to enable you to render its glyphs using syntax like `<i class="fas fa-camera"></i>`. However, the font is not actually used on the *Source Interface* (because it conflicts with Tor browser's behavior in the "Safest" security setting). Instead, we serve up static images rendered from the few glyphs that we use as icons.

The Font Awesome CSS is therefore likely wholly unnecessary. The issue here is to:
- Verify whether any part of the Font Awesome CSS is required for current *Source Interface* styling
- Strip the stylesheet as appropriate.

### PR DESCRIPTION
**Pull request by drewmassey**
## Status

Ready for review 
## Description of Changes

Fixes #4450.

Changes proposed in this pull request:
- removes font awesome library
- removes loading references to it in `_base.sass`

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.
_Nope_
### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make -C securedrop test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

### If you made changes to documentation:

- [ ] Doc linting (`make docs-lint`) passed locally

### CODE REVIEW

File: securedrop/sass/_base.sass [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 // LIBRARIES
 @import libraries/normalize
-@import libraries/font-awesome-iconography
```
Comment (redshiftzero) [inline securedrop/sass/_base.sass]: ah so what we want to do here is move this `font-awesome` import into `journalist.sass`. the reason being is that you are correct we do not use anything in the `font-awesome` library in the source interface but: we _do_ on the journalist interface (127.0.0.1:8081 in the dev env), e.g. these icons are all font awesome icons:

Screen Shot 2019-05-25 at 11 42 47 AM

**Comment by drewmassey** [comment]
Argh, sorry missing that in the code @redshiftzero. 43efa01 should resolve that. Thanks for bearing with me while I get situated.

**Comment by redshiftzero** [comment]
The test failures do seem to be legitimate here - functional tests also fail locally which one can do via the following:

```
$ cd securedrop
$ bin/dev-shell bin/run-test tests/functional/
```

And if you run `make dev` and log into the journalist interface, the font awesome icons don't appear (when you do that note that the globe icon in the language dropdown is a PNG, it's not rendered using font awesome). I think the culprit might be the order of SASS imports?

**Comment by drewmassey** [comment]
@redshiftzero I think this fixes it? The font awesome stuff shows up in journalist and all the tests pass locally.

**Pull request merged**